tests/integrations/raftstore/test_early_apply.rs

```diff
@@ -39,23 +38,23 @@ where
     let last_index = cluster.raft_local_state(1, 1).get_last_index();
     action(cluster);
     cluster.wait_last_index(1, 1, last_index + 1, Duration::from_secs(3));
-    let mut snaps = vec![(1, RocksSnapshot::new(cluster.get_raft_engine(1)))];
+    let mut rafts = vec![(1, cluster.get_raft_engine(1))];
```

Comment:
Does it perform as a snapshot?

Reply:
No, I'll fix it by loading the data into memory.